Library/Formula/docker-machine-driver-xhyve.rb

```diff
@@ -22,13 +22,21 @@ def install
     (buildpath/"gopath/src/github.com/zchee/docker-machine-driver-xhyve").install Dir["{*,.git,.gitignore}"]
 
     ENV["GOPATH"] = "#{buildpath}/gopath"
-
-    cd buildpath/"gopath/src/github.com/zchee/docker-machine-driver-xhyve" do
+    build_root = buildpath/"gopath/src/github.com/zchee/docker-machine-driver-xhyve"
+    cd build_root do
       if build.head?
         git_hash = `git rev-parse --short HEAD --quiet`.chomp
-        git_hash = " HEAD-#{git_hash}"
+        git_hash = "HEAD-#{git_hash}"
+        ENV["CGO_LDFLAGS"] = "#{build_root}/vendor/build/lib9p/lib9p.a -L#{build_root}/vendor/lib9p"
+        ENV["CGO_CFLAGS"] = "-I#{build_root}/vendor/lib9p"
+        mkdir "vendor/build"
```

Comment:
Ah, I see. TBH, I'm not sure why the build system is the way it is. It's my first go project I've dug into the source for, and one of my first attempts at making a brew formula. The directories are in .gitignore, so they're really not supposed to be in the source tree. Why the build doesn't doesn't make them itself is a mystery to me.

What would you like for me to do in this case? Would it be better to patch the Makefile instead of the formula? I've notified the upstream of the problem, so I'm wary that such a patch would fail to apply once the issue is solved.